### PR TITLE
docs: update links to changes for v0.47.9-ics-lsm

### PR DESCRIPTION
**Pull request by MSalopek**
Update deps to use latest version cosmos-sdk and ics with lsm:
* https://github.com/cosmos/cosmos-sdk/releases/tag/v0.47.9
* https://github.com/cosmos/interchain-security/releases/v3.3.2-lsm

### CODE REVIEW

File: .changelog/v15.0.0/bug-fixes/2919-bump-sdk.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 - Bump [cosmos-sdk](https://github.com/cosmos/cosmos-sdk) to
-  [v0.47.8-ics-lsm](https://github.com/cosmos/cosmos-sdk/tree/v0.47.8-ics-lsm).
+  [v0.47.9-ics-lsm](https://github.com/cosmos/cosmos-sdk/tree/v0.47.9-ics-lsm).
   This special branch of cosmos-sdk backports a series of fixes for issues found 
   during the [Oak Security audit of SDK 0.47](https://github.com/oak-security/audit-reports/blob/master/Cosmos%20SDK/2024-01-23%20Audit%20Report%20-%20Cosmos%20SDK%20v1.0.pdf).
-  ([\#2919](https://github.com/cosmos/gaia/pull/2919))
-  - Backport [sdk-#18146](https://github.com/cosmos/cosmos-sdk/pull/18146): Add denom check to reject denoms outside of those listed in `MinDeposit`. A new `MinDepositRatio` param is added (with a default value of `0.01`) and now deposits are required to be at least `MinDepositRatio*MinDeposit` to be accepted. ([sdk-#2919](https://github.com/cosmos/cosmos-sdk/pull/19312))
+  ([\#2954](https://github.com/cosmos/gaia/pull/2919))
```
Comment (mpoke) [inline .changelog/v15.0.0/bug-fixes/2919-bump-sdk.md]:
```suggestion
  ([\#2954](https://github.com/cosmos/gaia/pull/2954))
```

File: .changelog/v15.0.0/bug-fixes/2919-bump-sdk.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+  - Backport [sdk-#18146](https://github.com/cosmos/cosmos-sdk/pull/18146): Add denom check to reject denoms outside of those listed in `MinDeposit`. A new `MinDepositRatio` param is added (with a default value of `0.01`) and now deposits are required to be at least `MinDepositRatio*MinDeposit` to be accepted. ([sdk-#2954](https://github.com/cosmos/cosmos-sdk/pull/19312))
```
Comment (mpoke) [inline .changelog/v15.0.0/bug-fixes/2919-bump-sdk.md]:
```suggestion
  - Backport [sdk-#18146](https://github.com/cosmos/cosmos-sdk/pull/18146): Add denom check to reject denoms outside of those listed in `MinDeposit`. A new `MinDepositRatio` param is added (with a default value of `0.01`) and now deposits are required to be at least `MinDepositRatio*MinDeposit` to be accepted. ([sdk-#19312](https://github.com/cosmos/cosmos-sdk/pull/19312))
```

File: .changelog/v15.0.0/state-breaking/2954-bump-ics.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+- Bump [ICS](https://github.com/cosmos/interchain-security) to 
+  [v3.3.2-lsm](https://github.com/cosmos/interchain-security/releases/tag/v3.3.2-lsm) 
+  ([\#2954](https://github.com/cosmos/gaia/pull/2919))
```
Comment (mpoke) [inline .changelog/v15.0.0/state-breaking/2954-bump-ics.md]:
```suggestion
  ([\#2954](https://github.com/cosmos/gaia/pull/2954))
```

**Review by mpoke — CHANGES_REQUESTED** [review]
See my comments below. Also, rebase on main and then backport to release/v15.x.

**Review by sainoe — APPROVED** [review]

**Comment by mpoke** [comment]
Closing in favor of https://github.com/cosmos/gaia/pull/2957

**Pull request closed without merging**